src/train.py

```diff
@@ -248,6 +248,11 @@ def model_and_cost(config, is_train):
 
     for i in range(config['epochs']):
         # training: do not train first epoch, to see random weights behaviour
+
+        # modify the seed number on every epoch so that we get different patches
+        # from each track
+        np.random.seed(seed=config['seed'] + i)
```

Comment:
are you sure this is necessary? Subsequent calls to a random number generator should give different results every time?

Reply:
My intuition was exactly that, but empirically I found that the order was repeated on every epoch unless I added that line.
Now I'm looking into the problem again, and I believe that has to do with the parallel nature of the `Pescador` sampler. I found  [this issue](https://github.com/numpy/numpy/issues/9650#issuecomment-327139776) abut the problem.
My bet is that, as the sampling happens under a subthread for `Pesdacor`, it does not affect the `RandomState` of Numpy in the main thread. So unless the seed number is changed, in every epoch the sampler starts from the same point.